.cico/cico_openshift_e2e.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+set -ex
+
+# Setup to find nessasary data from cluster setup
+export CI="openshift"
+
+# Overrideable information
+export DEFAULT_INSTALLER_ASSETS_DIR=${DEFAULT_INSTALLER_ASSETS_DIR:-$(pwd)}
+export KUBEADMIN_USER=${KUBEADMIN_USER:-"kubeadmin"}
+export KUBEADMIN_PASSWORD_FILE=${KUBEADMIN_PASSWORD_FILE:-"${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeadmin-password"}
+
+# Exported to current env
+export KUBECONFIG=${KUBECONFIG:-"${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeconfig"}
+
+# Check if file with kubeadmin password exist
+if [ ! -f $KUBEADMIN_PASSWORD_FILE ]; then
+    echo "Could not find kubeadmin password file"
+    exit 1
+fi
+
+if [ ! -f $KUBECONFIG ]; then
+    echo "Could not find kubeconfig file"
+    exit 1
+fi
+
+# Get kubeadmin password from file
+export KUBEADMIN_PASSWORD=`cat $KUBEADMIN_PASSWORD_FILE`
```

Comment:
New sh syntax feature for me, I would write `$(cat ....)` and that simpler way works just fine )